source/options.tsx

```diff
@@ -57,6 +20,13 @@ function moveDisabledFeaturesToTop(): void {
 }
 
 function buildFeatureCheckbox({id, description, screenshot}: FeatureMeta): HTMLElement {
+	const descriptionElement = domify.one(description) as HTMLParagraphElement;
+	descriptionElement.className = 'description';
+
+	for (const a of select.all('a', descriptionElement)) {
+		applyToLink(a);
+	}
```

Comment:
We could also drop this, as it's not used anymore (most links are stripped anyway).

Unfortunately, `markdown-wasm` doesn't let us hook into the link rendering like `marked` or `markdown-it` do, to make URL shortening happen during the build.

Reply:
I think the only raw URL we have, where this is applied, is download-directory.github.io. If that’s the case, yes you can drop it too